docs/deployment/landing_page.rst

```diff
@@ -265,13 +265,15 @@ logging. However you still need to make sure no other assets get logged!
     SetEnvIf Request_URI "^/securedrop($|(\/.*))" dontlog
     CustomLog logs/access_log common env=!dontlog
 
-In nginx, logging can be disabled like so:
+In nginx, logging can be disabled by adding the following directives within the
+*Landing Page* ``location {}`` block:
```

Comment:
Do you think it's worth explicitly stating that they should create a location block for the landing page route?

Reply:
It's worth noting that the landing page docs say at the beginning:

> If the Landing Page is deployed on the same domain as another site, you might consider having some specific configuration (such as the security headers below) apply only to the /securedrop request URI. This can be done in Apache by the encapsulating these settings within a `<Location>` block, which can be defined similarly in nginx by using the `location {}` directive.